### PR TITLE
add optimizex verification & connect json conf

### PR DESCRIPTION
**Pull request by DinamickaDevelopment**

### CODE REVIEW

File: optimizex.io.connect.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "serviceName": "Optimizex Public Web Application",
+  "version": 1,
+  "logoUrl": "",
+  "description": "Enables a Domain to work with Optimizex",
+  "variableDescription": "HOST is the subdomain for the domain.",
+  "hostRequired": true,
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "%ox-cname-host%",
```
Comment (pawel-kow) [inline optimizex.io.connect.json:14]: if the subdomain is set with `host` parameter, then here I would expect `@` not a variable. Note that `@` will be replaced by the host to render the full hostname to write into the zone.

Comment (DinamickaDevelopment) [reply optimizex.io.connect.json:14]: Hi @pawel-kow
We need to set up this variable, individually for our clients, for example: for one client good is ox.site1.com, and for another mysybdomain.site2.com

Comment (pawel-kow) [reply optimizex.io.connect.json:14]: understood, in this case `@` is the right `host` value for the template.
In the application the `host` parameter will be used in place of `@`.

So, the examples would be called with `domain=site1.com` and `host=ox` or `domain=site2.com` and `host=mysubdomain` respectively.

Details on this mechanism in the specificaition https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#hostname-in-template

**Review by pawel-kow — CHANGES_REQUESTED** [review]
optimizex.io/connect template requires host change to `@`

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: optimizex.io.connect.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "providerId": "optimizex.io",
+  "providerName": "Optimizex",
+  "serviceId": "ox-connect",
```
[the remark this answers is not in the document]
Comment (DinamickaDevelopment) [reply optimizex.io.connect.json]: Hi @pawel-kow, it is already changed

**Review by pawel-kow — APPROVED** [review]
OK from my side now. Anyone from GD @wesbaker  or IONOS @lpanatau-ionos  wants to review as well?

File: optimizex.io.connect.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (DinamickaDevelopment) [reply optimizex.io.connect.json]: @pawel-kow tell me please how I can upload the optimizex.io icon?

[the remark this answers is not in the document]
Comment (DinamickaDevelopment) [reply optimizex.io.connect.json]: https://app.optimizex.io/icons/logo.svg

[the remark this answers is not in the document]
Comment (pawel-kow) [reply optimizex.io.connect.json]: @DinamickaDevelopment you need to host it yourself and add URL to the template.

**Comment by wesbaker** [comment]
I think you're looking for @wdbaker-godaddy

**Review by lpanatau-ionos — APPROVED** [review]
lgtm

**Comment by pawel-kow** [comment]
I guess this one was long enough in the review

**Pull request merged**